app-drawer-layout/app-drawer-layout.html

```diff
@@ -138,6 +138,14 @@
           value: '640px'
         },
 
+        /**
+         * True if the app-drawer-layout should fit the size of its container.
+         */
+        fullbleed: {
```

Comment:
This property is only needed for documentation. I guess the alternative is to add a getter to the prototype or just add a paragraph to the readme.  The only issue with that is that it won't be consistent.